libs/core/src/lib/progress-indicator/progress-indicator.component.html

```diff
@@ -0,0 +1,57 @@
+<div
+    #progressIndicator
+    class="fd-progress-indicator"
+    tabindex="-1"
+    role="progressbar"
+    [attr.aria-valuemin]="valueMin"
+    [attr.aria-valuenow]="valueNow"
+    [attr.aria-valuemax]="valueMax"
+    [attr.aria-valuetext]="valueText"
+    [attr.aria-label]="valueText"
+    [ngClass]="state ? 'fd-progress-indicator--' + state : ''"
```

Comment:
Angular recommends use `[class]` https://angular.io/guide/attribute-binding#binding-to-the-class-attribute

Reply:
Some more details https://github.com/angular/angular/issues/40623